### PR TITLE
Update static_type_checking.md for tox4

### PR DESCRIPTION
**Pull request by kristapratico**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: doc/dev/static_type_checking.md [hunk 1]
```diff
@@ -175,17 +175,17 @@ version of the type checker ships. All client libraries in the Python SDK repo a
...
+`pip install tox`
 
 ### Run mypy
 
 mypy is currently pinned to version [1.0.0](https://pypi.org/project/mypy/1.0.0/).
 
 To run mypy on your library, run the tox mypy env at the package level:
 
-`.../azure-sdk-for-python/sdk/textanalytics/azure-ai-textanalytics>tox -e mypy -c ../../../eng/tox/tox.ini`
+`.../azure-sdk-for-python/sdk/textanalytics/azure-ai-textanalytics>tox run -e mypy -c ../../../eng/tox/tox.ini --root sdk/textanalytics/azure-ai-textanalytics`
```
Comment (pvaneck) [inline doc/dev/static_type_checking.md:188]: These should all probably be `--root .` since the command is being run from inside the project folder.

Comment (kristapratico) [reply doc/dev/static_type_checking.md:188]: good call!

Comment (kristapratico) [inline doc/dev/static_type_checking.md:188]:
```suggestion
`.../azure-sdk-for-python/sdk/textanalytics/azure-ai-textanalytics>tox run -e mypy -c ../../../eng/tox/tox.ini --root .`
```

File: doc/dev/static_type_checking.md [hunk 2]
```diff
@@ -217,7 +217,7 @@ Note that pyright requires that node is installed. The command-line [wrapper pac
 
 To run pyright on your library, run the tox pyright env at the package level:
 
-`.../azure-sdk-for-python/sdk/textanalytics/azure-ai-textanalytics>tox -e pyright -c ../../../eng/tox/tox.ini`
+`.../azure-sdk-for-python/sdk/textanalytics/azure-ai-textanalytics>tox run -e pyright -c ../../../eng/tox/tox.ini --root sdk/textanalytics/azure-ai-textanalytics`
```
Comment (kristapratico) [inline doc/dev/static_type_checking.md:220]:
```suggestion
`.../azure-sdk-for-python/sdk/textanalytics/azure-ai-textanalytics>tox run -e pyright -c ../../../eng/tox/tox.ini --root .`
```

File: doc/dev/static_type_checking.md [hunk 3]
```diff
@@ -249,7 +249,7 @@ from the code in the PR vs. the latest release on PyPi.
 
 To run verifytypes on your library, run the tox verifytypes env at the package level:
 
-`.../azure-sdk-for-python/sdk/textanalytics/azure-ai-textanalytics>tox -e verifytypes -c ../../../eng/tox/tox.ini`
+`.../azure-sdk-for-python/sdk/textanalytics/azure-ai-textanalytics>tox run -e verifytypes -c ../../../eng/tox/tox.ini --root sdk/textanalytics/azure-ai-textanalytics`
```
Comment (kristapratico) [inline doc/dev/static_type_checking.md:252]:
```suggestion
`.../azure-sdk-for-python/sdk/textanalytics/azure-ai-textanalytics>tox run -e verifytypes -c ../../../eng/tox/tox.ini --root .`
```

**Comment by kristapratico** [comment]
Going to close this, looks the docs updates were made in the PR that updated tox 😄

**Pull request closed without merging**